.github/workflows/test_production_function.yaml

```diff
@@ -183,6 +180,10 @@ jobs:
               {
                 $m365 = $attribute
               }
+              elseif($key.ToLower() -eq "emails")
+              {
```

Comment:
Style (see https://github.com/PoshCode/PowerShellPracticeAndStyle/blob/master/Style-Guide/Code-Layout-and-Formatting.md#one-true-brace-style)

Reply:
Bruh.  Can we not get hung up on things that don't matter, please?